sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -159,11 +165,52 @@ public partial class RedisEnterprisePrivateEndpointConnectionResource : Azure.Re
 }
 namespace Azure.ResourceManager.RedisEnterprise.Models
 {
+    public partial class Capability
+    {
+        internal Capability() { }
+        public string Name { get { throw null; } }
+        public bool? Value { get { throw null; } }
+    }
+    public partial class ClusterPropertiesEncryptionCustomerManagedKeyEncryption
+    {
+        public ClusterPropertiesEncryptionCustomerManagedKeyEncryption() { }
+        public Azure.ResourceManager.RedisEnterprise.Models.ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity KeyEncryptionKeyIdentity { get { throw null; } set { } }
+        public System.Uri KeyEncryptionKeyUri { get { throw null; } set { } }
+    }
+    public partial class ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity
+    {
+        public ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity() { }
+        public Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType? IdentityType { get { throw null; } set { } }
+        public string UserAssignedIdentityResourceId { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CmkIdentityType : System.IEquatable<Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType>
```

Comment:
Could we rename this type to `RedisEnterpriseCmkIdentityType`?
And for consistency, please also consider unify to use only `CustomerManagedKey` or its abbr `Cmk` in all the related models

Reply:
Renamed and using CustomerManagedKey